.github/workflows/iroha2-add-label.yml

```
@@ -0,0 +1,15 @@
+ name: Add Iroha2 label 
```

Could you remove space ' '?

```suggestion
name: Add Iroha2 label 
```